package.json

```diff
@@ -33,7 +33,8 @@
     "testonly": "babel-node ./node_modules/.bin/_mocha $npm_package_options_mocha",
     "lint": "eslint src",
     "check": "flow check",
-    "build": "rm -rf lib/* && babel src --ignore __tests__ --out-dir lib",
+    "build": "rm -rf lib/* && babel src --ignore __tests__ --out-dir lib && cp package.json lib/ && npm run build:flow",
```

Comment:
Well there are two problems: one is that it is copying the `package.json` unnecessarily (IIRC it's used in the `graphql` NPM package release process, where Travis CI cuts the releases), and the other is that it is copying it into `lib` instead of `dist`, which wouldn't be right even if we did have a Travis-based release process in this repo.